('shanghai', (None, 5))
('ant', (None, 4))
('cjg', (None, 4))
('cgp', (None, 4))
('cgm', (None, 4))
('fgx', (None, 4))
('glu', (None, 4))
('hiy', (None, 4))
('jid', (None, 4))
('jas', (None, 4))
('jqi', (None, 4))
('jns', (None, 4))
('kus', (None, 4))
('lus', (None, 4))
('mng', (None, 4))
('nnh', (None, 4))
('pnu', (None, 4))
('sgh', (None, 4))
('sgj', (None, 4))
('szh', (None, 4))
('wja', (None, 4))
('zhq', (None, 4))
('zxj', (None, 4))
('pvg', (None, 0))
('sha', (None, 0))
('zspd', (None, 1))
('zsss', (None, 1))
('zssl', (None, 1))
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 42, in <module>
    main()
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 20, in main
    trie = create_trie(locations)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 39, in create_trie
    return RecordTrie('<HH', code_id_type_tuples)
  File "marisa_trie.pyx", line 649, in marisa_trie.RecordTrie.__init__ (src/marisa_trie.cpp:11840)
  File "marisa_trie.pyx", line 598, in marisa_trie._UnpackTrie.__init__ (src/marisa_trie.cpp:10663)
  File "marisa_trie.pyx", line 422, in marisa_trie.BytesTrie.__init__ (src/marisa_trie.cpp:7670)
  File "marisa_trie.pyx", line 124, in marisa_trie._Trie._build (src/marisa_trie.cpp:2655)
  File "marisa_trie.pyx", line 421, in genexpr (src/marisa_trie.cpp:7475)
  File "marisa_trie.pyx", line 597, in genexpr (src/marisa_trie.cpp:10540)
  File "marisa_trie.pyx", line 655, in marisa_trie.RecordTrie._pack (src/marisa_trie.cpp:11983)
struct.error: required argument is not an integer
('shanghai', (None, 5))
('ant', (None, 4))
('cjg', (None, 4))
('cgp', (None, 4))
('cgm', (None, 4))
('fgx', (None, 4))
('glu', (None, 4))
('hiy', (None, 4))
('jid', (None, 4))
('jas', (None, 4))
('jqi', (None, 4))
('jns', (None, 4))
('kus', (None, 4))
('lus', (None, 4))
('mng', (None, 4))
('nnh', (None, 4))
('pnu', (None, 4))
('sgh', (None, 4))
('sgj', (None, 4))
('szh', (None, 4))
('wja', (None, 4))
('zhq', (None, 4))
('zxj', (None, 4))
('pvg', (None, 0))
('sha', (None, 0))
('zspd', (None, 1))
('zsss', (None, 1))
('zssl', (None, 1))
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 42, in <module>
    main()
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 20, in main
    trie = create_trie(locations)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 39, in create_trie
    return RecordTrie('<HH', code_id_type_tuples)
  File "marisa_trie.pyx", line 649, in marisa_trie.RecordTrie.__init__ (src/marisa_trie.cpp:11840)
  File "marisa_trie.pyx", line 598, in marisa_trie._UnpackTrie.__init__ (src/marisa_trie.cpp:10663)
  File "marisa_trie.pyx", line 422, in marisa_trie.BytesTrie.__init__ (src/marisa_trie.cpp:7670)
  File "marisa_trie.pyx", line 124, in marisa_trie._Trie._build (src/marisa_trie.cpp:2655)
  File "marisa_trie.pyx", line 421, in genexpr (src/marisa_trie.cpp:7475)
  File "marisa_trie.pyx", line 597, in genexpr (src/marisa_trie.cpp:10540)
  File "marisa_trie.pyx", line 655, in marisa_trie.RecordTrie._pack (src/marisa_trie.cpp:11983)
struct.error: required argument is not an integer
('shanghai', (None, 5))
('ant', (None, 4))
('cjg', (None, 4))
('cgp', (None, 4))
('cgm', (None, 4))
('fgx', (None, 4))
('glu', (None, 4))
('hiy', (None, 4))
('jid', (None, 4))
('jas', (None, 4))
('jqi', (None, 4))
('jns', (None, 4))
('kus', (None, 4))
('lus', (None, 4))
('mng', (None, 4))
('nnh', (None, 4))
('pnu', (None, 4))
('sgh', (None, 4))
('sgj', (None, 4))
('szh', (None, 4))
('wja', (None, 4))
('zhq', (None, 4))
('zxj', (None, 4))
('pvg', (None, 0))
('sha', (None, 0))
('zspd', (None, 1))
('zsss', (None, 1))
('zssl', (None, 1))
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 42, in <module>
    main()
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 20, in main
    trie = create_trie(locations)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 39, in create_trie
    return RecordTrie('<HH', code_id_type_tuples)
  File "marisa_trie.pyx", line 649, in marisa_trie.RecordTrie.__init__ (src/marisa_trie.cpp:11840)
  File "marisa_trie.pyx", line 598, in marisa_trie._UnpackTrie.__init__ (src/marisa_trie.cpp:10663)
  File "marisa_trie.pyx", line 422, in marisa_trie.BytesTrie.__init__ (src/marisa_trie.cpp:7670)
  File "marisa_trie.pyx", line 124, in marisa_trie._Trie._build (src/marisa_trie.cpp:2655)
  File "marisa_trie.pyx", line 421, in genexpr (src/marisa_trie.cpp:7475)
  File "marisa_trie.pyx", line 597, in genexpr (src/marisa_trie.cpp:10540)
  File "marisa_trie.pyx", line 655, in marisa_trie.RecordTrie._pack (src/marisa_trie.cpp:11983)
struct.error: required argument is not an integer
('shanghai', (None, 5))
('ant', (None, 4))
('cjg', (None, 4))
('cgp', (None, 4))
('cgm', (None, 4))
('fgx', (None, 4))
('glu', (None, 4))
('hiy', (None, 4))
('jid', (None, 4))
('jas', (None, 4))
('jqi', (None, 4))
('jns', (None, 4))
('kus', (None, 4))
('lus', (None, 4))
('mng', (None, 4))
('nnh', (None, 4))
('pnu', (None, 4))
('sgh', (None, 4))
('sgj', (None, 4))
('szh', (None, 4))
('wja', (None, 4))
('zhq', (None, 4))
('zxj', (None, 4))
('pvg', (None, 0))
('sha', (None, 0))
('zspd', (None, 1))
('zsss', (None, 1))
('zssl', (None, 1))
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 42, in <module>
    main()
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 20, in main
    trie = create_trie(locations)
  File "/data/rdns-parse/src/find_and_evaluate/create_trie.py", line 39, in create_trie
    return RecordTrie('<HH', code_id_type_tuples)
  File "marisa_trie.pyx", line 649, in marisa_trie.RecordTrie.__init__ (src/marisa_trie.cpp:11840)
  File "marisa_trie.pyx", line 598, in marisa_trie._UnpackTrie.__init__ (src/marisa_trie.cpp:10663)
  File "marisa_trie.pyx", line 422, in marisa_trie.BytesTrie.__init__ (src/marisa_trie.cpp:7670)
  File "marisa_trie.pyx", line 124, in marisa_trie._Trie._build (src/marisa_trie.cpp:2655)
  File "marisa_trie.pyx", line 421, in genexpr (src/marisa_trie.cpp:7475)
  File "marisa_trie.pyx", line 597, in genexpr (src/marisa_trie.cpp:10540)
  File "marisa_trie.pyx", line 655, in marisa_trie.RecordTrie._pack (src/marisa_trie.cpp:11983)
struct.error: required argument is not an integer
